src/RNCamera.js

```diff
@@ -534,6 +542,14 @@ export default class Camera extends React.Component<PropsType, StateType> {
     return isAuthorized ? CameraStatus.READY : CameraStatus.NOT_AUTHORIZED;
   };
 
+  getRecordAudioPermissionStatus = (): RecordAudioPermissionStatus => {
```

Comment:
we do not need this method, the default value of `this.state.recordAudioPermissionStatus` `RecordAudioPermissionStatusEnum.PENDING_AUTHORIZATION`